tests/main/security-seccomp/task.yaml

```diff
@@ -44,8 +44,10 @@ prepare: |
 restore: |
     #shellcheck source=tests/lib/dirs.sh
     . "$TESTSLIB/dirs.sh"
-    mv -f "$SRC".orig "$SRC"
-    snapd.tool exec snap-seccomp compile "$SRC" "$BIN"
+    if [ -e "$SRC".orig ]; then
```

Comment:
nitpick, you could adjust the test to use the cleanup/defer tool, but this is ok as-is